src/data/move.ts

```diff
@@ -570,6 +570,17 @@ export abstract class Move implements Localizable {
     return this;
   }
 
+  /**
+   * Sets the {@linkcode MoveFlags.G_MAX_MOVE} for the move
+   * and {@linkcode moveTarget} to NEAR_ENEMY (g-max moves cannot target allies)
+   * @returns The {@linkcode Move} that called this function
+   */
+  gmaxMove(): this {
```

Comment:
NIT: Shouldn't it be:
```suggestion
  gMaxMove(): this {
```
Since it's also `G_MAX_MOVE`?